### PR TITLE
[16.0][MIG] auth_api_key_group

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

**Review by jbaudoux — APPROVED** [review]

File: auth_api_key_group/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Alpha-red.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Alpha
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--auth-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-auth/tree/15.0/auth_api_key_group
```
Comment (IT-Ideas) [inline auth_api_key_group/README.rst:17]: Shouldn't the README be updated? I think about the links that should use 16.0 and no more 15.0.

File: auth_api_key_group/i18n/auth_api_key_group.pot [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* auth_api_key_group
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0\n"
```
Comment (IT-Ideas) [inline auth_api_key_group/i18n/auth_api_key_group.pot:7]: 16.0?

File: auth_api_key_group/static/description/index.html [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+!! This file is generated by oca-gen-addon-readme !!
+!! changes will be overwritten.                   !!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Alpha" src="https://img.shields.io/badge/maturity-Alpha-red.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/lgpl-3.0-standalone.html"><img alt="License: LGPL-3" src="https://img.shields.io/badge/licence-LGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/server-auth/tree/15.0/auth_api_key_group"><img alt="OCA/server-auth" src="https://img.shields.io/badge/github-OCA%2Fserver--auth-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/server-auth-15-0/server-auth-15-0-auth_api_key_group"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/251/15.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
```
Comment (IT-Ideas) [inline auth_api_key_group/static/description/index.html:370]: Shouldn't the index.html be updated? I think about the links that should use 16.0 and no more 15.0.

**Review by IT-Ideas — COMMENTED** [review]
Thanks!

File: auth_api_key_group/README.rst (hunk 1 above)
Comment (jbaudoux) [reply auth_api_key_group/README.rst:17]: No need, it is regenerated automatically by the merge bot

File: auth_api_key_group/static/description/index.html (hunk 3 above)
Comment (jbaudoux) [reply auth_api_key_group/static/description/index.html:370]: No need, it is regenerated automatically by the merge bot

**Review by sonhd91 — APPROVED** [review]

**Review by simahawk — APPROVED** [review]
LG

@sbejaoui I'm curious... I fail to understand why you keep adding ` - ` to commit messages :)
There's no need to separate tags or other parts w/ that char.

**Pull request merged**